datasets/common_voice/README.md

```diff
@@ -61,9 +61,9 @@ languages:
 - uk
 - vi
 - vot
-- zh-CN
-- zh-HK
-- zh-TW
+- zh-Hans-CN
+- zh-Hans-HK
+- zh-Hant-TW
```

Comment:
We should keep  zh-CN, zh-HK and zh-TW since they are valid codes according to [BCP-47](https://tools.ietf.org/search/bcp47)

The `Hans` subtag means that the text uses the Simplified Chinese script

Therefore I'd suggest to add these three to our list of supported languages

Reply:
sounds good, will add them to `datasets`!